cinema_service/settings.py

```
@@ -86,8 +86,12 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.environ.get("POSTGRES_DB", None),
+        "USER": os.environ.get("POSTGRES_USER", None),
+        "PASSWORD": os.environ.get("POSTGRES_PASSWORD", None),
+        "HOST": os.environ.get("POSTGRES_HOST", None),
+        "PORT": os.environ.get("POSTGRES_PORT", None),
```

Issue: Using os.environ.get(..., None) for required database environment variables can result in None values if the environment variable is missing, causing unclear errors at runtime. According to robust dockerization practices, you should use os.environ[...] instead, so the application fails fast with a clear error if any required variable is missing.